lib/cocoapods/target/aggregate_target.rb

```diff
@@ -285,38 +291,38 @@ def podfile_dir_relative_path
     #         the user project.
     #
     def xcconfig_relative_path(config_name)
-      relative_to_srcroot(xcconfig_path(config_name)).to_s
```

Comment:
can you please keep this as a call to `relative_to_srcroot` ?

Reply:
I want to remove the all method about the srcroot. All path should not relative the srcroot.

Reply:
but this directory is still relative to the SRCROOT

Reply:
This is a separate PR for #7781.

The #7781 will cause the directory may be relative the SRCROOT.

Reply:
You could potentially wait until this PR is merged, and then update #7781 to include this change

Reply:
@amorde I separate the #7781 to this PR and #7879 , and closed #7781 .